python/tests/test_extend.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+from hyperon import *
+
+class ExtendTest(unittest.TestCase):
+
+    def test_extend(self):
+        metta = MeTTa()
+        self.assertEqual(
+            metta.run('''
+              !(extend-py! extention)
```

Comment:
Two items here:
1. I would suggest getting string with a module name as an argument instead of symbol.
2. Why `extend-py` and not `import-py`?

Reply:
> I would suggest getting string with a module name as an argument instead of symbol.

It works both with strings and symbols. I simply use `repr` if you noticed. The issue was that I first tried to debug this functionality on importing `stdlib`, which introduces strings. For this case, it is not possible to use strings at all. Also, I noticed that `import stdlib` without quotes looks natural in Python. And third, we discussed the possibility that strings will be implemented in Rust, and it is not guaranteed that Python strings will continue working. Strings can be used in this examples (although I don't think it is more useful than using a symbol), but I'm quite strongly against removing support for changing the current behavior with `repr` (unless something equivalent but safer is proposed).

Reply:
> Why `extend-py` and not `import-py`?

Well, I wanted to emphasize the current difference between importing expressions by `import` and extending MeTTa itself by `extend-py` (which will not just add grounded atoms, but also may tweak the runner behavior itself). However, since we've discussed that `extend-py` will also be able to act as `import` (add expressions to a space, although the question is what space to use and whether `extend-py` will have a space as its argument), then it may make sense to rename it. My conclusion is: if `extend-py` gets an additional argument (space) and its signature will be the same as of `import` (and it will import expressions to the space), then it can be renamed. While the signature remains different, it should not be renamed.

Reply:
> I simply use repr if you noticed.

Yes, I have noticed and it is the reason why I am asking. Because `repr` allows using anything which is representable as a string instead of module name, and it is not clean from my perspective. After you comment I realized that strings cannot be used because they are supported by stdlib, but then I would use symbols only. But it is my personal preference probably.

Reply:
> difference between importing expressions by import and extending MeTTa itself by extend-py

I don't see why current PR doesn't include adding atoms into the MeTTa atomspace?